src/modules/scrap/parser/parser.service.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpException, HttpStatus, Injectable, Logger } from "@nestjs/common";
+import * as csv from 'fast-csv';
+
+
+@Injectable()
+export class ParserService {
+  private readonly logger = new Logger(ParserService.name);
+  constructor(
+  ) { }
+
+  // check csv valid
+  async processCsvFile(fileBuffer: Buffer) {
```

Comment:
1. How about passing `fileBuffer` to the constructor instead?
2. How about renaming this function to `isValidCsvFile` since it returns a boolean:

```suggestion
  async isValidCsvFile(fileBuffer: Buffer) {
```